timescale-forge/maintenance.md

```diff
@@ -0,0 +1,52 @@
+# Maintenance
+On Timescale Forge, software updates are handled automatically by us, and you do
+not need to perform any actions.
+
+Non-critical software updates are applied during a maintenance window that you
+can define to suit your workload.  If we detect a security vulnerability that
+affects you, we might need to perform maintenance outside of the scheduled
+maintenance window.
+
+<highlight type="important">
+After a maintenance update, the DNS name remains the same, but the IP address
+it points to changes.
```

Comment:
```suggestion
it points to most likely changes.
```

It almost always changes AFAIK but I don't think that's a guarantee we want to give.

Reply:
```suggestion
it points to often changes.
```
??